bbpower/compsep.py

```diff
@@ -589,6 +589,20 @@ def lnprob(self, par):
         like = -0.5 * np.dot(dx, np.dot(self.invcov, dx))
         return prior + like
 
+    def lnlike(self, par):
+        """
+        Likelihood without priors. 
+        """
+        params = self.params.build_params(par)
```

Comment:
To avoid repeating code too much, could you replace the analogous lines in `lnprob` with just a call to this new `lnlike` function?